rasa/core/tracker_store.py

```diff
@@ -33,20 +39,25 @@ def __init__(
 
     @staticmethod
     def find_tracker_store(domain, store=None, event_broker=None):
```

Comment:
If you don't mind it would be great to also add types to the functions. We are doing it step by step. E.g.
```
    def find_tracker_store(
        domain: Domain,
        store: Optional[EndpointConfig] = None,
        event_broker: Optional[EventChannel] = None,
    ) -> "TrackerStore":
```

Reply:
I added these return type hints to the base class and the DynamoDb tracker store.